modules/core/src/main/java/org/apache/synapse/unittest/TestingAgent.java

```diff
@@ -0,0 +1,396 @@
+/*
+ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
small formatting issue here
let's fix the same in other locations as well

Reply:
Fixed in https://github.com/wso2/wso2-synapse/pull/1452